addons/mail_plugin/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+mail_plugin.access_res_partner_iap,access_res_partner_iap,mail_plugin.model_res_partner_iap,base.group_system,1,1,1,1
```

Comment:
Why not `0,0,0,0`? The model doesn't seem exposed through a view / UI element, and sudo mode completely bypasses ACL.

Reply:
@xmo-odoo Generally I think we prefer to set 1,1,1,1 for the group system (I had the comment on an other task for a similar model which was not in a view, and so I did it here)

Reply:
I also think admins should be able to globally see everything. Note that we could think of adding views for this information in debug mode under IAP menus in Technical. What do you think ?

Reply:
Good idea, it will also allow to debug things :)